v22.1/ui-jobs-page.md

```diff
@@ -60,12 +60,15 @@ Execution Count | Number of times the job was executed.
 Status | Description
 -------|------------
 `PENDING` | Job is created but has not started running.
-`PAUSED` | Job is [paused](pause-job.html).
+`PAUSED` | Job is [paused]({{ link_prefix }}pause-job.html).
+`PAUSE-REQUESTED` | A request has been issued to pause the job. The status will move to `PAUSED` when the node running the job registers the request.
 `FAILED` | Job failed to complete.
 `SUCCEEDED` | Job successfully completed.
-`CANCELED` | Job was [cancelled](cancel-job.html).
+`CANCELED` | Job was [canceled]({{ link_prefix }}cancel-job.html).
```

Comment:
Here's a fun one: CRL is a US company, and US English prefers `cancelled`, but our chosen status spelling is `CANCELED`, the UK spelling! So: we must do just as you've done here and use the UK common form everywhere in our US docs when talking about these statuses!

Reply:
Weird... I actually thought it was the other way round. That UK preferred doubled. I am so past remembering UK spelling at this point. My whole editorial and writing career has been in the US! 

Re the caps... I do agree, though the reason I kept it the same, was because on the Jobs Page the UI shows them as capped. In the `SHOW JOBS` output we see them as lowercased. I'll leave the PR as-is for now, as a result. But, I'm also open to any strong feelings you have on that.

Reply:
Oops, you're right, I switched them up! Completely ignore me!!

For caps: No strong preference here either. If the command outputs lowercase, I'd go with that as the standard. But also fine: as is, definitely.